### PR TITLE
improve apo page

### PR DESCRIPTION
**Pull request by fiatjaf**

### CODE REVIEW

File: content/proposals/apo.md [hunk 1]
```diff
@@ -34,6 +38,8 @@ This proposal has been specified in
 
 ## Use Cases
 
-APO was specifically proposed to bring about Eltoo, but can be used to achieve various other use
-cases. However,
-
+APO was specifically proposed to bring about [Eltoo](/use-cases/eltoo), but can be used to achieve various other use
+cases, such as [Statechains](/use-cases/statechains) and [Spacechains](/use-cases/spacechains). It
+can also be used to inefficiently emulate a [CTV](/proposals/ctv)-like covenant and all its
+use-cases, but the efficiency loss makes these use cases much less compelling than the 3 proposals
```
Comment (stevenroose) [inline content/proposals/apo.md:44]: What do you mean with the "3 proposals above"?

Comment (fiatjaf) [reply content/proposals/apo.md:44]: Eltoo, Spacechains and Statechains.

But yeah, this is weird.

I was just trying to imagine what you meant with the cryptic "However," that ended the text before. What were you going to say?

Comment (stevenroose) [reply content/proposals/apo.md:44]: Ah yeah I made of lot of this pages in a hurry, I think I was going to say that in order to use APO for other things like cong contr or stuff, it's actually pretty roundabout and inefficient. Also fragile cuz you need to keep signatures etc.

**Comment by stevenroose** [comment]
Let me take this and rephrase a bit and merge.

**Comment by stevenroose** [comment]
Does this make sense to you? a4ad57aec324ec2c2ef6975cee14d512d3e4dc83

**Comment by stevenroose** [comment]
Please close this MR by way of ack, or make comments and I'll try to address.

**Comment by fiatjaf** [comment]
I like my version more than your changes. What are "real covenants"? Why did you remove the mentions to Statechains and Spacechains?, these are use cases that are better served by APO than by CTV, that's why I included them.

Of course TXHASH also works to implement them and is probably even better than APO, but it wouldn't hurt to leave these two things mentioned here as a tribute to the fact that they were originally proposed with APO in mind.

Anyway, I'm fine with the way you wrote it.

**Comment by stevenroose** [comment]
But APO can be used to do anything that CTV can do "in some way", so I didn't understand why mention statechains and spacechains specifically. The way you'd use APO for spacechains seems pretty hackish to me if I understand what you mean. For statechains maybe it makes more sense than templating opcodes.

**Comment by stevenroose** [comment]
Reworded again, leaning more on your phrasing :)

**Comment by fiatjaf** [comment]
Thank you, I like it more now.

Weirdly, CTV is bad for Spacechains. It delivers a thing that can work but is fragile and may cause transaction fees to be wasted and hurt the incentives of BMM. APO is better because it allows including outputs after the fact in the presigned transactions (APO+SINGLE+ANYONECANPAY). I think TXHASH and others should be good for Spacechains too, but I didn't look very closely.

For Statechains, the mechanism is the same of Eltoo/LN Simmetry, so as far as I understand it requires APO or CSFS? Or can we do that with TXHASH?

**Pull request closed without merging**